src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const newObj = {};
```

Comment:
not the best name🤔

[NAMING] - use proper names for variables - if you want to name variable `result`(`arr`, `obj`), pick a better one :)